src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/ProjectsController.java

```diff
@@ -60,6 +60,19 @@ public String updateProcurementProjectName(@PathVariable("procID") final Integer
     return Constants.OK_MSG;
   }
 
+  @PutMapping("/{procID}/close")
```

Comment:
Should probably wrap the response in `StringValueResponse` - this forces it into a valid JSON response (i.e. quoted) - otherwise the unquoted response is technically invalid JSON. See example usage in `EventController.publishEvent(...)`

Reply:
done